ldap-connector-demo/src_hd/com/axonivy/connector/ldap/connector/demo/query/queryProcess.mod

```diff
@@ -0,0 +1,160 @@
+[Ivy]
+17D6ADD32E68D727 9.3.1 #module
+>Proto >Proto Collection #zClass
+qs0 queryProcess Big #zClass
+qs0 RD #cInfo
+qs0 #process
+qs0 @AnnotationInP-0n ai ai #zField
+qs0 @TextInP .type .type #zField
+qs0 @TextInP .processKind .processKind #zField
+qs0 @TextInP .xml .xml #zField
+qs0 @TextInP .responsibility .responsibility #zField
+qs0 @UdInit f0 '' #zField
+qs0 @UdProcessEnd f1 '' #zField
+qs0 @PushWFArc f2 '' #zField
+qs0 @UdEvent f3 '' #zField
+qs0 @UdExitEnd f4 '' #zField
+qs0 @PushWFArc f5 '' #zField
+qs0 @UdProcessEnd f7 '' #zField
+qs0 @CallSub f9 '' #zField
+qs0 @UdEvent f6 '' #zField
+qs0 @PushWFArc f8 '' #zField
+qs0 @PushWFArc f10 '' #zField
+qs0 @ErrorBoundaryEvent f11 '' #zField
+qs0 @GridStep f12 '' #zField
+qs0 @PushWFArc f13 '' #zField
+qs0 @PushWFArc f14 '' #zField
+>Proto qs0 qs0 queryProcess #zField
+qs0 f0 guid 17D6ADD32EBA05F1 #txt
+qs0 f0 method start() #txt
+qs0 f0 inParameterDecl '<> param;' #txt
+qs0 f0 inParameterMapAction 'out.filter="(objectClass=*)";
+out.Password=ivy.var.LdapConnector_Password;
+out.provider=ivy.var.LdapConnector_Provider;
+out.rootObject="CN=Users,DC=zugtstdomain,DC=wan";
+out.url=ivy.var.LdapConnector_Url;
+out.user=ivy.var.LdapConnector_Username;
+' #txt
+qs0 f0 outParameterDecl '<> result;' #txt
+qs0 f0 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>start()</name>
+    </language>
+</elementInfo>
+' #txt
+qs0 f0 83 51 26 26 -16 15 #rect
+qs0 f1 211 51 26 26 0 12 #rect
+qs0 f2 109 64 211 64 #arcP
+qs0 f3 guid 17D6ADD32F076B1D #txt
+qs0 f3 actionTable 'out=in;
+' #txt
+qs0 f3 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>close</name>
+    </language>
+</elementInfo>
+' #txt
+qs0 f3 83 147 26 26 -15 15 #rect
+qs0 f4 211 147 26 26 0 12 #rect
+qs0 f5 109 160 211 160 #arcP
+qs0 f7 411 211 26 26 0 12 #rect
+qs0 f9 processCall query:call(com.axonivy.connector.ldap.LdapQuery) #txt
+qs0 f9 requestActionDecl '<com.axonivy.connector.ldap.LdapQuery ldapQuery> param;' #txt
+qs0 f9 requestActionCode 'import javax.naming.directory.SearchControls;
+import com.axonivy.connector.ldap.LdapQuery;
+import com.axonivy.connector.ldap.util.JndiConfig;
+
+ivy.var.set("LdapConnector.Username", in.user);
```

Comment:
you should not set these variables here. because this will write somewhere in app.yaml in your Axon Ivy Designer installation and will be persistent.